src/backend/util2.c

```diff
@@ -54,6 +54,8 @@ void util_assert(const char *file, int line)
 {
     fflush(stdout);
     printf("Internal error: %s %d\n",file,line);
+    printf("This is a compiler bug, please report it via "
+        "https://issues.dlang.org/enter_bug.cgi\n");
```

Comment:
But still, it should be major, not critical. Critical would be things like silently generating bad code, which is much worse than an assert.